.circleci/config.yml

```diff
@@ -140,20 +140,6 @@ commands:
                   key: v2-golden-images-{{ .Branch }}-<< parameters.regression_system >>-<< parameters.regression_color >>-<< parameters.regression_scale >>-<< parameters.regression_dir >>-{{ epoch }}
 
 jobs:
-    commitlint:
```

Comment:
Question: it seems like we don't need this now? With changesets, the conventional commit syntax is a lot less critical. We still have this running on commits locally, do we also need it in CI?

Reply:
Good point... I think this was just an extra check in case someone’s local setup wasn’t working as expected, but it does feel redundant to me. I’m fine removing it, and if we ever spot a suspicious commit message in the git log, we can always add it back 👍